testsuite/MDAnalysisTests/test_selections.py

```diff
@@ -61,6 +61,9 @@ def _write_selection(self, **kwargs):
         g.write_selection(self.namedfile, **kwargs)
         return g
 
+    def test_write_bad_mode(self):
```

Comment:
Or you could just open another PR with a more obviously-wrong value in an additional test--testing for a valid standard library open-based flag and an invalid one would serve two slightly different purposes.